### PR TITLE
enhanced build to work without any prerequisites on the target machine

### PR DESCRIPTION
**Pull request by teichsta**
Signed-off-by: Thomas Eichstädt-Engelen <thomas@openhab.org>

### CODE REVIEW

File: extensions/ui/org.eclipse.smarthome.ui.paperui/pom.xml [hunk 1]
```diff
@@ -16,38 +16,53 @@
 		<bundle.namespace>org.eclipse.smarthome.ui.paperui</bundle.namespace>
 	</properties>
 
-	<groupId>org.eclipse.smarthome.ui</groupId>
```
Comment (sja) [inline extensions/ui/org.eclipse.smarthome.ui.paperui/pom.xml]: Why is this removed?

Comment (teichsta) [inline extensions/ui/org.eclipse.smarthome.ui.paperui/pom.xml]: it get's inherited from the parent pom …
